decoder.go

```diff
@@ -188,6 +188,7 @@ func (d *Decoder) decode(v reflect.Value, path string, parts []pathPart, values
 	if conv == nil && t.Kind() == reflect.Slice {
 		var items []reflect.Value
 		elemT := t.Elem()
+		//elemP := elemT
```

Comment:
Didn't notice that. I cherry-picked the previous commits as-is. I'll remove this now.